python/lsst/afw/geom/ellipses/__init__.py

```diff
@@ -23,23 +23,20 @@
 """lsst.afw.geom.ellipses
 """
 from __future__ import absolute_import
-from builtins import object
 from .ellipsesLib import *
-import weakref
-import numpy
 
-Separable = {
-    (Distortion, DeterminantRadius):SeparableDistortionDeterminantRadius,
-    (Distortion, TraceRadius):SeparableDistortionTraceRadius,
-    (Distortion, LogDeterminantRadius):SeparableDistortionLogDeterminantRadius,
-    (Distortion, LogTraceRadius):SeparableDistortionLogTraceRadius,
-    (ConformalShear, DeterminantRadius):SeparableConformalShearDeterminantRadius,
-    (ConformalShear, TraceRadius):SeparableConformalShearTraceRadius,
-    (ConformalShear, LogDeterminantRadius):SeparableConformalShearLogDeterminantRadius,
-    (ConformalShear, LogTraceRadius):SeparableConformalShearLogTraceRadius
-}
+#Separable = {
+#    (Distortion, DeterminantRadius):SeparableDistortionDeterminantRadius,
+#    (Distortion, TraceRadius):SeparableDistortionTraceRadius,
+#    (Distortion, LogDeterminantRadius):SeparableDistortionLogDeterminantRadius,
+#    (Distortion, LogTraceRadius):SeparableDistortionLogTraceRadius,
+#    (ConformalShear, DeterminantRadius):SeparableConformalShearDeterminantRadius,
+#    (ConformalShear, TraceRadius):SeparableConformalShearTraceRadius,
+#    (ConformalShear, LogDeterminantRadius):SeparableConformalShearLogDeterminantRadius,
+#    (ConformalShear, LogTraceRadius):SeparableConformalShearLogTraceRadius
+#}
```

Comment:
Yes, that's fine, but they deserve a code comment. Please? Note that in some cases the justification has been "because we don't need it yet" but in others it's because "it doesn't work yet". That is important information that needs to be in the code as a comment, not just on github.